opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -219,6 +219,18 @@ message Histogram {
   // aggregation_temporality describes if the aggregator reports delta changes
   // since last report time, or cumulative changes since a fixed start time.
   AggregationTemporality aggregation_temporality = 2;
+
+  enum SumType {
+    // The histogram sum is a monotonic value and will not decrease.
+    // Note: This is the default for histograms if unspecified.
+    MONOTONIC = 0;
```

Comment:
Should we avoid 0 as we may not know if it's set or missing?

Maybe...
NONE = 0
MONOTONIC = 1
NON_MONOTONIC = 2
UNKNOWN_SUM = 3

Reply:
Backwards-commpatiblity means previous users would NOT be specifying this value, which defaults to 0.  So we should treat that as MONOTONIC.

Reply:
Is it that before we had sum as monotonic?

Reply:
Yes, sum was monotonic before

Reply:
From here, that is not clear:

```
  // Note: Sum should only be filled out when measuring non-negative discrete
  // events, and is assumed to be monotonic over the values of these events.
  // Negative events *can* be recorded, but sum should not be filled out when
  // doing so.  This is specifically to enforce compatibility w/ OpenMetrics,
  // see: https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#histogram
```

Reply:
See the linked bug.  Basically we can't *not* fill out a sum, so due to the phrasing, we're always providing a sum and it's always monotonic.